hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/builder/TopicBuilder.java

```diff
@@ -65,6 +66,10 @@ public static TopicBuilder topic(TopicName topicName) {
         return new TopicBuilder(topicName);
     }
 
+    public static TopicBuilder topic(ExtensionContext context) {
```

Comment:
I see that in [this PR](https://github.com/allegro/hermes/pull/1785) we added following method:
``` java
public static TopicBuilder topicWithRandomName() {
    return topic("group" + sequence.incrementAndGet(), "topic" + sequence.incrementAndGet());
}
```
Both create topic with random name, so which one do we want to use?